keyboards/dztech/dz65rgb/keymaps/drootz/config.h

```diff
@@ -0,0 +1,49 @@
+#define LEADER_PER_KEY_TIMING
```

Comment:
```suggestion
#pragma once

#define LEADER_PER_KEY_TIMING
```

Reply:
Thanks! Addressed in PR #8199 Feedback Commit #1 - c0be53d